### PR TITLE
fix verbosity

### LINKED ISSUE #38: ENH: verbosity levels
pipelines can have 3 verbosity levels

1. `verbose=0`: nipype terminal_output set to 'none' and AFNI commands passed with -quiet option when available
2. `verbose=1`: nipype terminal_output set to 'allatone' and AFNI commands passed with default verbosity
3. `verbose=2`: nipype terminal_output set to 'allatone' and AFNI commands passed with -verb option when available

### PR DESCRIPTION
**Pull request by salma1601**
fix #38

### CODE REVIEW

**Comment by salma1601** [comment]
@nadkarni-na I finished the verbosity change. can you have a look please ?
Also please tell me which PRs you need to be merged first, because there will be conflicts between struct.py versions.

**Comment by nadkarni-na** [comment]
@salma1601 it would be good to finish this soon. I could try to go through the conflict resolver though it might be faster for you to do the changes at your end. another option is for me to fork the branch but I think that's overkill

**Comment by salma1601** [comment]
Ok, I am on it

File: sammba/registration/struct.py [hunk 1]
```diff
@@ -232,7 +241,6 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
                               **brain_masking_unifize_kwargs)
         brain_masking_in_files.append(out_unifize.outputs.out_file)
     
-    # brain mask creation
```
Comment (nadkarni-na) [inline sammba/registration/struct.py]: Is this deletion intentional?

File: sammba/registration/struct.py [hunk 2]
```diff
@@ -413,8 +431,9 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
             one_pass=True,
             weight=out_mask_tool.outputs.out_file,
             out_file=fname_presuffix(shift_rotated_head_file,
-                                     suffix='_affine'))
-        # matrix concatenation
```
Comment (nadkarni-na) [inline sammba/registration/struct.py]: same here, can the comment be kept?

**Comment by salma1601** [comment]
It was a mistake, I am rewriting them.

**Comment by salma1601** [comment]
merged.

**Comment by nadkarni-na** [comment]
thanks! using it now

**Pull request merged**